UnitTest failure with Python 3.9

Hello Podhmo,

Your package is failling with Python 3.9:

```
__________________ LazyArgumentsTests.test_with_actual_types2 __________________

self = <prestring.tests.test_lazy_object.LazyArgumentsTests testMethod=test_with_actual_types2>

    def test_with_actual_types2(self):
        try:
            import typing as t
    
            target = self._makeOne(
                ["x", "y", "z"],
                types={
                    "x": int,
                    "y": t.Optional[int],
                    "z": t.Sequence[t.Optional[int]],
                },
            )
>           self.assertEqual(
                str(target),
                "x: int, y: 'typing.Union[int, NoneType]', z: 'typing.Sequence[typing.Union[int, NoneType]]'",
            )
E           AssertionError: "x: int, y: 'typing.Optional[int]', z: 'typing.Sequence[typin[13 chars]t]]'" != "x: int, y: 'typing.Union[int, NoneType]', z: 'typing.Sequenc[27 chars]e]]'"
E           - x: int, y: 'typing.Optional[int]', z: 'typing.Sequence[typing.Optional[int]]'
E           ?                    ^^^   --                                   ^^^   --
E           + x: int, y: 'typing.Union[int, NoneType]', z: 'typing.Sequence[typing.Union[int, NoneType]]'
E           ?                    ^^       ++++++++++                               ^^       ++++++++++
```

Looks harmless but you might assert Optional[int] or Union[int, NoneType] which is at the end, the same.

Best regards, Adam.

--- a/prestring/tests/test_lazy_object.py
+++ b/prestring/tests/test_lazy_object.py
@@ -29,23 +29,20 @@ class LazyArgumentsTests(unittest.TestCase):
         self.assertEqual(str(target), "x: int, y: bool, *")
 
     def test_with_actual_types2(self):
-        try:
-            import typing as t
+        import typing as t
 
-            target = self._makeOne(
-                ["x", "y", "z"],
-                types={
-                    "x": int,
-                    "y": t.Optional[int],
-                    "z": t.Sequence[t.Optional[int]],
-                },
-            )
-            self.assertEqual(
-                str(target),
-                "x: int, y: 'typing.Union[int, NoneType]', z: 'typing.Sequence[typing.Union[int, NoneType]]'",
-            )
-        except ImportError:
-            pass
+        target = self._makeOne(
+            ["x", "y", "z"],
+            types={
+                "x": int,
+                "y": t.Optional[int],
+                "z": t.Sequence[t.Optional[int]],
+            },
+        )
+        self.assertEqual(
+            str(target),
+            "x: int, y: 'typing.Union[int, NoneType]', z: 'typing.Sequence[typing.Union[int, NoneType]]'",
+        )
 
 
 @test_target("prestring:LazyKeywords")

--- a/prestring/tests/test_lazy_object.py
+++ b/prestring/tests/test_lazy_object.py
@@ -39,10 +39,20 @@ class LazyArgumentsTests(unittest.TestCase):
                 "z": t.Sequence[t.Optional[int]],
             },
         )
-        self.assertEqual(
-            str(target),
-            "x: int, y: 'typing.Union[int, NoneType]', z: 'typing.Sequence[typing.Union[int, NoneType]]'",
-        )
+
+        # TODO: fix, this is work-around
+        import sys
+
+        if (3, 9) > sys.version_info:
+            self.assertEqual(
+                str(target),
+                "x: int, y: 'typing.Union[int, NoneType]', z: 'typing.Sequence[typing.Union[int, NoneType]]'",
+            )
+        else:
+            self.assertEqual(
+                str(target),
+                "x: int, y: 'typing.Optional[int]', z: 'typing.Sequence[typing.Optional[int]]'",
+            )
 
 
 @test_target("prestring:LazyKeywords")

